installation.sh

```diff
@@ -181,8 +191,22 @@ if [ -n "$values" ]; then
   values_command=" -f $values"
 fi
 
+alert_manager_url_command=""
+if [ -n "$alert_manager_url" ]; then
+  alert_manager_url_command=" --set globalConfig.alertmanager_url=$alert_manager_url"
+fi
+
+prometheus_org_id_command=""
+echo "Prometheus org id: $prometheus_org_id"
```

Comment:
may be not required 🤔 
someone can define using Helm config. there can be too many cli args, so we can skip some if we feel that is not very common sceanrio

Reply:
also its partial ? because itys not setting same on opencost

Reply:
**Ignore, got opencost changes